modules/vectorfieldvisualization/src/processors/datageneration/seedpointsfrommask.cpp

```diff
@@ -93,24 +95,31 @@ void SeedPointsFromMask::process() {
             util::IndexMapper3D index(dim);
             vec3 invDim = vec3(1.0f) / vec3(dim);
 
+            std::function<vec3(vec3)> noTransform = [](vec3 p) -> vec3 { return p; };
```

Comment:
I redesigned it now. 
I needed it before since I wanted to select between this and the next lambda to avoid an if statement inside the lambda or in the for loop.